npm/src/upload/upload-form.ts

```diff
@@ -175,6 +175,8 @@ export class UploadForm {
   ) {
     this.formElement.addEventListener("submit", (ev) => {
       ev.preventDefault()
+
+      this.disableButtonsAndDropzone()
```

Comment:
Let's have a chat about this in our 1-1 today. I think this change will help in most circumstances, but there are some subtle edge-cases that this might not catch.

Reply:
It would also be good to have a test to make sure the form only gets submitted once, but let's chat about the implementation first.